homeassistant/components/onkyo/config_flow.py

```diff
@@ -67,6 +69,40 @@ class OnkyoConfigFlow(ConfigFlow, domain=DOMAIN):
     _receiver_info: ReceiverInfo
     _discovered_infos: dict[str, ReceiverInfo]
 
+    async def async_step_ssdp(
+        self, discovery_info: ssdp.SsdpServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle flow initialized by SSDP discovery."""
+        host = urlparse(discovery_info.ssdp_location).hostname
+        _LOGGER.debug("Config flow start ssdp auto discovery: %s", host)
+
+        if not host:
+            _LOGGER.error("SSDP host is None: %s", discovery_info)
+            return self.async_abort(reason="no_host")
+
+        try:
+            info = await async_interview(str(host))
+        except Exception:
+            _LOGGER.exception("Unexpected exception interviewing host %s", host)
+            return self.async_abort(reason="host_info_error")
+
+        if info is None:
+            _LOGGER.error("SSDP info is none: %s", host)
+            return self.async_abort(reason="host_info_none")
```

Comment:
This is not translated properly. Also I'm not sure if we even should invent a completely new reason for aborting here.

Reply:
Ah I missed how the strings were working, I updated to use the existing error strings.